.buildkite/auditbeat/scripts/package-step.sh

```diff
@@ -18,7 +18,7 @@ if are_files_changed "$changeset"; then
       - label: ":ubuntu: Packaging Linux X86"
         key: "package-linux-x86"
         env:
-          PLATFORMS: "+all linux/amd64 linux/arm64 windows/amd64 darwin/amd64 darwin/arm64"
+          PLATFORMS: "+all linux/amd64 linux/arm64 windows/amd64 darwin/amd64"
```

Comment:
**nit**: will this env variable be refactored in the future? If so, could a link to the relevant GitHub issue help avoid forgetting this?

Reply:
I think it would be great to define that variable in one common place as it is defined in Jenkins. I'm only not sure we have to file an issue for that. I think it's just a part of future changes when we start aggregating our scripts.
@oakrizan @v1v WDYT?